content/en/tracing/runtime_metrics/dotnet.md

```diff
@@ -38,6 +38,12 @@ The following metrics are collected by default after enabling .NET metrics.
 
 Along with displaying these metrics in your APM Service Page, Datadog provides a [default .NET Runtime Dashboard][6] with the `service` and `runtime-id` tags that are applied to these metrics.
 
+##  Additional permissions for IIS
+
+On .NET Framework, metrics are collected using performance counters. Users in non-interactive logon sessions (that includes IIS application pool accounts and some service accounts) must be added to the **Performance Monitoring Users** group to access counter data.
+
+IIS application pools use special accounts that do not appear in the list of users. To add them to the Performance Monitoring Users group, look for `IIS APPPOOL\<name of the pool>`. For instance, the user for the DefaultAppPool would be `IIS APPPOOL\DefaultAppPool`.
```

Comment:
Is it worth being more explicit exactly how to do this, e.g. _how_ do I add a user to a user group? If that's outside the scope of our documentation, could we link somewhere?